chore(docs): update health check documentation (#1677)
questdb/questdb.io: docs/guides/importing-data.md

### Configure `COPY`

> - Enable `COPY` and [configure](/docs/reference/configuration/#csv-import)
>   `COPY` directories to suit your server.

Reviewer: ```suggestion
- Enable `COPY` and [configure](/docs/reference/configuration/#csv-import)
  the `COPY` directories to suit your server.
```

Author: Done in 700c38836a40040a09ee9c7cc7136bfa976b152e

## FAQ

<details>
  <summary>COPY on a table with symbol columns is very slow. How can I speed it up?</summary>
<p>

> QuestDB uses `256` as the default symbol capacity. In case if the number of
> distinct symbol column values exceeds this default significantly, the `COPY`
> performance will suffer. Make sure that you specify symbol capacities when

Reviewer: ```suggestion
QuestDB uses `256` as the default symbol capacity. If the number of
distinct symbol values exceeds this default significantly, the `COPY`
performance will suffer. Make sure that you specify symbol capacities when
```

Author: Done in 700c38836a40040a09ee9c7cc7136bfa976b152e

> performance will suffer. Make sure that you specify symbol capacities when
> pre-creating the table before running the `COPY` command. Here is an example:

Reviewer: ```suggestion
performance will suffer. Make sure that you specify a symbol capacity when
creating the table before running the `COPY` command. 

Here is an example:
```

```questdb-sql
CREATE TABLE table_name (
  ts TIMESTAMP,
  sym SYMBOL CAPACITY 100000
) TIMESTAMP(ts) PARTITION BY DAY;
```

> Refer to [symbol type documentation](/docs/concept/symbol/#symbol-columns) for

Reviewer: ```suggestion
Refer to the [symbol type documentation](/docs/concept/symbol/#symbol-columns) for
```

> more information on configuring symbol capacity.

Reviewer: ```suggestion
more information on configuring the symbol capacity.
```
